src/message/MessageList.js

```diff
@@ -17,6 +17,11 @@ const styles = StyleSheet.create({
 export default class MessageList extends React.PureComponent {
   autoScrollToBottom = false;
 
+  constructor(props) {
+    super();
+    this.state = { actionSheetButtons: ['', ''] };
```

Comment:
We should use `state` sparingly.
I don't like the API of this action-sheet component at all. It almost forces us to use state, but it is not a good approach.